### PR TITLE
improve env settings (see PR qc atlas ui)

### PR DESCRIPTION
**Pull request by manuwei**
* improve naming of env variables / ports etc.

### CODE REVIEW

**Comment by manuwei** [comment]
TODO: Change the ports in docu after merging this PR (frontend available at 80 instead of the development port)

File: docker-compose.yml [hunk 1]
```diff
@@ -27,16 +27,16 @@ services:
...
+      # from the container itself we have to specify localhost here
       # and **not** the container name
-      ALTAS_URL: localhost
-      ATLAS_PORT: 8080
-      PATTERNPEDIA_URL: localhost
-      PATTERNPEDIA_PORT: 8082
-      NISQ_ANALYZER_URL: localhost
+      QC_ATLAS_HOST_NAME: localhost
+      QC_ATLAS_PORT: 8080
+      PATTERN_ATLAS_HOST_NAME: test
```
Comment (wederbn) [inline docker-compose.yml]: Should be localhost?

Comment (manuwei) [reply docker-compose.yml]: Absolutely. I'll fix that

**Review by wederbn — APPROVED** [review]
LGTM

**Review by salmma — APPROVED** [review]
LGTM

**Pull request merged**